pycc/local.py

```diff
@@ -44,7 +48,9 @@ class Local(object):
 
     def __init__(self, local, C, nfzc, no, nv, H, cutoff,
             core_cut=5E-2,
-            lindep_cut=1E-6):
+            lindep_cut=1E-6,
+            e_conv=1e-12,
+            r_conv=1e-12):
```

Comment:
Is there a reason for the defaults in `Local()` to be different than those here?

Reply:
I don't quite follow. the convergence thresholds are set to be quite tight since the MP2 is (generally) not a problem to converge (and there's currently no way to push a different convergence criterion here through the Coupled Cluster code, since that convergence isn't set by the user until _after_ the `Local` class object is generated). the `lindep_cut` and `core_cut` were lifted from [Hampel & Werner 1996](doi.org/10.1063/1.471289) and the Psi3 code, respectively.